.buildkite/coverage.yaml

```diff
@@ -10,4 +10,5 @@ steps:
     env:
       DOCKER_IMAGE: "gcr.io/opensourcecoin/osrank-build@sha256:00e4c02a7391457031c2f3d3a548e92aa44fd2341de08918968040fe9f2793eb"
     agents:
-      docker: "true"
+      platform: linux
+      production: true
```

Comment:
Not sure if this has to be quoted (boolean is a type in YAML). Seems to not not work, tho

Reply:
Yeah, not sure either -- it's confusing because in the Buildkite UI, neither `linux` or `true` are quoted, but it's also true that the previous incantation of the command was quoted. Let me try quoting both..